impl/src/bazel.rs

```diff
@@ -35,6 +35,8 @@ static SUPPORTED_PLATFORM_TRIPLES: &'static [&'static str] = &[
   "x86_64-apple-darwin",
   "x86_64-pc-windows-gnu",
   "x86_64-unknown-linux-gnu",
+  "i686-pc-windows-msvc",
```

Comment:
These cannot be added until they're added in https://github.com/bazelbuild/rules_rust/blob/master/rust/platform/platform.bzl

Raze currently relies on the platforms defined in `rules_rust` to do it's configuration switching. If you feel these targets should be supported, can you first get a pull request merged there? Then I'd ask that you update the revision of `rules_rust` specified in `smoke-test.sh` since that ultimately determines what gets output in the examples.

Reply:
Sorry, I left some comments in the commit but should have posted them here as well. This is not intended to be merged as-is: https://github.com/google/cargo-raze/pull/245/commits/efe960419f3aead5b88edcfb97b023abf219597b